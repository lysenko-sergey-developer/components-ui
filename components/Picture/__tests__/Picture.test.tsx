import * as React from 'react'
import { shallow } from 'enzyme'
import { Picture } from '..'

describe('Picture test cases', () => {
  it('Picture was success rendered!', () => {
  const component = shallow(<Picture/>)

  expect(component).toMatchSnapshot()
  })
})
