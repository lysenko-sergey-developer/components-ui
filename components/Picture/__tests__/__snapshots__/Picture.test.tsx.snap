// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Picture test cases Picture was success rendered! 1`] = `ShallowWrapper {}`;
